utils/umaApi.ts

```diff
@@ -11,32 +22,142 @@ const baseOptions = {
   method: "POST",
 };
 
-const baseUrl =
-  process.env.NEXT_PUBLIC_UMA_API_URL || "https://prod.api.umaproject.org";
+const API_URLS: Record<ChainId, string> = {
+  1: "https://prod.api.umaproject.org",
```

Comment:
Should we pass these as ENV vars?

Reply:
yea for sure, theres an example of using different chain env vars by just appending a `_chainid` to the end of the env.

For example we could probably just keep this `NEXT_PUBLIC_UMA_API_URL` as mainnet but also add
`NEXT_PUBLIC_UMA_API_URL_137` for polygon and `NEXT_PUBLIC_UMA_API_URL_1` also for mainnet, etc

Reply:
We could, but they're public anyway and I think its annoying / dangerous to have to set a bunch of env variables. So I'm in favor of hardcoding

Reply:
I was thinking only from the perspective of testing API versions other than the ones used in production. It's easier to edit an .env file than looking in the source code, but I agree in this case it's not a big deal

Reply:
nbd, will change it to use env variables and use the hardcoded URLs as fallback